services/core-api/app/templates/email/mine_party_appt/minespace_new_eor_email.html

```diff
@@ -0,0 +1,151 @@
+<html>
```

Comment:
I had made this template in error- currently we have no use case to send this email to minespace users. However, we have a mockup for it. Can either delete this file or keep it in case we use it in the future.

Reply:
I feel like there's a real possibility that this is something that will be needed for minespace users once we get into the user management side of things there, so I wouldn't mind keeping this!